src/utilities/conditionals.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Determines wether or not district seats should be distributed for a given year.
+ *
+ * @param year The year to check
+ */
+export function distributeDistrictSeats(year: number): boolean {
```

Comment:
Maybe better to have `shouldDistributeDistrictSeats` -- right now the function name indicates that it _does_ distribute district seats. I find it's nice to have the first segmented word immediately make you think of a boolean.

Reply:
That is indeed a better name. Fixed.